src/components/proofs/Card.tsx

```diff
@@ -1,27 +1,29 @@
 import { useSnapshot } from 'valtio'
 import Card from 'components/Card'
-import CardContainer from 'components/proofs/CardContainer'
 import ConnectAccount from 'components/proofs/ConnectAccount'
 import List from 'components/proofs/List'
 import LoadingCard from 'components/proofs/LoadingCard'
 import WalletStore from 'stores/WalletStore'
-import ZkProofHintWhenLg from 'components/proofs/ZkProofHintWhenLg'
 
 export default function () {
   const { account, walletLoading } = useSnapshot(WalletStore)
 
   return (
-    <CardContainer>
-      <Card paddingType="normal" color="accent" shadow useAppStyles nospace>
-        {walletLoading ? (
-          <LoadingCard />
-        ) : account ? (
-          <List />
-        ) : (
-          <ConnectAccount />
-        )}
-      </Card>
-      <ZkProofHintWhenLg />
-    </CardContainer>
+    <Card
+      paddingType="normal"
+      color="accent"
+      shadow
+      useAppStyles
+      nospace
+      onlyWrap
```

Comment:
These two may lead to weird behavior. Did you test and checked everything?

Reply:
I've tested moderately and I haven't had any issues with any element positioning, but i'll test a bit more just to make sure